source/1nce_zero_touch_provisioning.c

```diff
@@ -203,164 +230,168 @@ uint8_t nce_onboard( char ** pThingName,
-    TlsTransportStatus_t xNetworkStatus = nce_connect( &xNetworkContext );
-
-    if( TLS_TRANSPORT_SUCCESS != xNetworkStatus )
-    {
-        LogError( ( "Failed to connect to 1NCE server." ) );
-        return status;
-    }
-
-    /* Build onboarding request. */
-    char packetToSent[ 100 ];
-
-    memset( packetToSent, '\0', 100 * sizeof( char ) );
-    sprintf( packetToSent, "GET /device-api/onboarding HTTP/1.1\r\n"
-                           "Host: %s\r\n"
-                           "Accept: text/csv\r\n\r\n", ONBOARDING_ENDPOINT );
-    LogInfo( ( "Send onboarding request:\r\n%.*s",
-               strlen( packetToSent ),
-               packetToSent ) );
-
-    /* Send onboarding request. */
-    int32_t sentBytes = TLS_FreeRTOS_send( &xNetworkContext,
-                                           &packetToSent,
-                                           strlen( packetToSent ) );
-
-    configASSERT( sentBytes > 0 );
-
-    if( sentBytes <= 0 )
-    {
-        LogError( ( "Failed to send onboarding request." ) );
-        return status;
-    }
-
-    /* Receive onboarding response. */
-    int32_t recvBytes = TLS_FreeRTOS_recv( &xNetworkContext,
-                                           &PART[ 0 ],
-                                           RECV_BUFFER_LEN );
-
-    if( recvBytes < 0 )
-    {
-        LogError( ( "Failed to receive onboarding response." ) );
-        return status;
-    }
-
-    LogDebug( ( "Received raw response: %d bytes.", recvBytes ) );
-    LogDebug( ( "\r\n%.*s", recvBytes, PART ) );
-    strcat( completeResponse,
-            strstr( PART, "Express\r\n\r\n\"" ) + strlen( "Express\r\n\r\n\"" ) );
-    memset( PART, ( int8_t ) '\0', sizeof( PART ) );
-
-    while( recvBytes == RECV_BUFFER_LEN )
-    {
-        recvBytes = TLS_FreeRTOS_recv( &xNetworkContext,
-                                       &PART[ 0 ],
-                                       RECV_BUFFER_LEN );
-
-        if( recvBytes < 0 )
-        {
-            LogError( ( "Failed to receive onboarding response." ) );
-            return status;
-        }
-
-        LogDebug( ( "Received raw response: %d bytes.", strlen( PART ) ) );
-        LogDebug( ( "\r\n%.*s", strlen( PART ), PART ) );
-        strcat( completeResponse, PART );
-        memset( PART, ( int8_t ) '\0', sizeof( PART ) );
-    }
-
-    LogInfo( ( " Onboarding response is received." ) );
-
-    /* Disconnect onboarding TLS connection. */
-    TLS_FreeRTOS_Disconnect( &xNetworkContext );
-
-    /* Re-initialize MQTT connection information with onboarding information. */
-    status = nceReinitConnParams( completeResponse,
-                                  pThingName,
-                                  pEndpoint,
-                                  pExampleTopic,
-                                  pRootCA,
-                                  pClientCert,
-                                  pPrvKey );
-    return status;
+	uint8_t status = EXIT_FAILURE;
```

Comment:
seems like the indent has been shift, could you help revert it? thanks

Reply:
Thanks for your comment. 
The Indentation has been fixed.